extensions/shared/paid-blocks/components.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { SVG, Icon, Circle } from '@wordpress/components';
+
+const paidIconIndicator = ( { cx = 21, cy = 3, r = 4, fill = '#e34c84' } ) => {
+	return (
+		<SVG>
+			<Circle cx={ cx } cy={ cy } r={ r } fill="#fff" />
```

Comment:
This seems odd — couldn't it be just a `stroke` attribute on a single circle?

Reply:
Yeah sure! We will go for a static approach though that uses a `stroke` here https://github.com/Automattic/jetpack/pull/16509/files#diff-47e791c7d16dade6976183ded5310c88